projects/k8s/dbt_project.yml

```diff
@@ -0,0 +1,54 @@
+
+# Name your project! Project names should contain only lowercase characters
+# and underscores. A good package name should reflect your organization's
+# name or the intended use of these models
+name: 'k8s'
+version: '1.0.0'
+
+# This setting configures which "profile" dbt uses for this project.
+profile: 'k8s'
```

Comment:
The profile should be the same as the db name  app_dev_k8s, and it should be the same name for the project. k8s in it self is a bit cryptic.